source/MaterialXTest/MaterialXRenderGlsl/RenderGlsl.cpp

```diff
@@ -101,6 +101,7 @@ void GlslShaderRenderTester::registerLights(mx::DocumentPtr document,
     REQUIRE(envIrradiance);
     _lightHandler->setEnvRadianceMap(envRadiance);
     _lightHandler->setEnvIrradianceMap(envIrradiance);
+    _lightHandler->setEnvSamples(256);
```

Comment:
I'm a bit worried about how long this will take to run with a lot of files.

Could be add a fidelity option into the `_options.mtlx` file which controls these kind of parameters. e.g. a "high" vs "low" quality setting ? "high" can include turning on shadow maps for GLSL for instance.

Reply:
I wouldn't expect a real-world performance impact from these extra samples, since they're performed on the GPU, and our render tests currently spend most of their time in file I/O and shader compilation.  For the future, I think you're right that a high/low option for quality could be valuable, especially if we end up running render tests on a software rasterizer one day.  But it's likely worthwhile to wait until we have a real-world use case for the "low" quality mode, and we can then tailor it specifically for the needs of that case.  For now, adding shadows wouldn't have a visual impact, since we're restricted to rendering spheres in order to match testrender, though this limitation is likely to be lifted in the future.